README.md

```diff
@@ -1,132 +1,135 @@
-* Define an  animate function that calls: `renderer.render(scene, camera)`
+@almarklein: A few suggestions and feature discussions
+- fast draws (57 FPS @ 30M vertices from 1k lines on a . . .) (@almarklein: what
+  specs does your laptop have? Also, do we have a polygon count benchmark?)
+- (approximate) order-independent transparency (OIT). (is this implemented now?)
+- Custom post-processing. (do the examples for this need updating?)
+- Easy shader templating
+- built atop [WGPU](https://github.com/pygfx/wgpu-py) (>= OpenGL)
+- semantically similar to TreeJS (WorldObjects, decoupled rendering backend, ...)
 
+And more! Check out our [feature
+demos](https://pygfx.readthedocs.io/en/latest/_gallery/index.html) in the docs.
 
-## On world objects, materials, and geometry
+## License
 
-There are a few different world object classes. The class defines
-(semantically) the kind of object being drawn, e.g. `Line`, `Image`,
-`Mesh`, `Volume`. World objects have a position and orientation in the
-scene, and can have children (other world objects), creating a tree.
-World objects can also have a geometry and/or a material.
+Pygfx is licensed under the [BSD 2-Clause "Simplified" License](LICENSE). This means:
 
-The geometry of an object defines its base data, usually per-vertex
-attributes such as positions, normals, and texture coordinates. There
-are several pre-defined geometries, most of which simply define certain
-3D shapes.
+- :white_check_mark: It is free (and open source) forever. :cupid:
+- :white_check_mark: You _can_ use it commercially.
+- :white_check_mark: You _can_ distribute it and freely make changes.
+- :x: You _can not_ hold us accountable the results of using pygfx.
 
-The material of an object defines how an object is rendered. Usually
-each WorldObject class has one or more materials associated with it.
-E.g. a line can be drawn solid, segmented or with arrows. A volume can
-be rendered as a slice, MIP, or something else.
+## Contributing
+We use a very similar setup to many other open-source libraries in the python
+ecosystem. If you have contributed to open-source in the past, a lot of this
+should look familiar.
 
+### GitHub Workflow
+We use a PR-based workflow. Each contributor has his/her own fork of the project
+inside of which he/she maintains a branch containing the changes that should be
+incorporated into the project. These changes are presented as a [pull request
+(PR)](https://github.com/pygfx/pygfx/pulls) on GitHub where we discuss them,
+review code, and eventually merge the work. 
 
-## Installation
+When you begin working on pygfx you would typically do something like:
 
 ```bash
-pip install -U pygfx
-```
-Or, to get the latest from GitHub:
-```bash
-pip install -U https://github.com/pygfx/pygfx/archive/main.zip
+# Click the Fork button on GitHub and navigate to your fork
+git clone <address_of_your_fork>
+cd pygfx
+git remote add upstream git@github.com:pygfx/pygfx.git
+pip install -e .[dev,docs,examples]
```

Comment:
Are contributors smart enough to use a venv themselves?

Reply:
I agree there are many existing ones, but I don't know a good one (never searched). If you know one I'd be happy to link it. It's mainly for people that are new to open-source work to have something easy to get them started.

Alternatively, we could drop the contributing section entirely or shrink it. Depends on how eager we are to attract external contributions.

Reply:
This one perhaps? https://docs.github.com/en/get-started/quickstart/github-flow

Reply:
Sounds good. I will still keep a snippet on how to get a dev install, because that is specific to pygfx (i.e. which optional dependencies to install).